verkletree/examples/test_verkle_tree.rs

```diff
@@ -9,7 +9,13 @@ fn main() {
     let (circuit, root) = create_verkle_tree_proof(leaf, indices);
 
     let k = 10;
+
     let mut prover = VerkleTreeProver::new(k, circuit, true);
+    let (params, vk) = prover.get_verifier_params();
```

Comment:
is it possible to init the verifier without init the prover ? cz current get_verifier_params makes them couple to each other